site/src/pages/components/invokers.explainer.mdx

```diff
@@ -382,6 +382,8 @@ that this list is ordered and higher rules take precedence:
 | `<details>`     | `'close'`             | If the `<details>` is `open`, then close it                                          |
 | `<select>`      | `'showPicker'`        | Call `.showPicker()` on the invokee                                                  |
 | `<input>`       | `'showPicker'`        | Call `.showPicker()` on the invokee                                                  |
+| `<input>`       | `'stepUp'`            | Call `.stepUp()` on the invokee                                                      |
+| `<input>`       | `'stepDown'`          | Call `.stepDown()` on the invokee                                                    |
```

Comment:
Can we change these to make clear it's just aimed at number inputs?

Reply:
Ah if type="number" won't fit without just removing spaces maybe leave for now. We can adjust it once other prs that refactor this make it in.